pkg/slo-controller/config/constants.go

```diff
@@ -17,10 +17,11 @@
 package config
 
 const (
-	ConfigNameSpace            = "koordinator-system"
-	SLOCtrlConfigMap           = "slo-controller-config"
-	ColocationConfigKey        = "colocation-config"
-	ResourceThresholdConfigKey = "resource-threshold-config"
-	ResourceQoSConfigKey       = "resource-qos-config"
-	CPUBurstConfigKey          = "cpu-burst-config"
+	ConfigNameSpace                  = "koordinator-system"
+	SLOCtrlConfigMap                 = "slo-controller-config"
+	ColocationConfigKey              = "colocation-config"
+	ResourceThresholdConfigKey       = "resource-threshold-config"
+	ResourceQoSConfigKey             = "resource-qos-config"
+	CPUBurstConfigKey                = "cpu-burst-config"
+	NodeMetricCollectPolicyConfigKey = "node-metric-collect-policy-config"
```

Comment:
there're too many config keys right now. maybe we should consider simplify them.

Reply:
Yes. The current configuration synchronization mechanism does not have flexible scalability, we really should have a better way